Homework_4.py

```diff
@@ -5,10 +5,10 @@
 
 def type_checker():
     enter = input("Enter your text ")
-    if(enter.isdigit() == True):
-        if (float(enter) % 2 == 0):
+    if enter.isdigit():
+        if int(enter) % 2:
```

Comment:
This condition is equal to the next one and you will never get the right output for even number.
Since, for example, 3 % 1 returns 1 your condition will be transformed first to `if 1` and then to `if True`, what means that this condition will be True for value 3, which is not an even number. To fix it you can add `if int(enter) % 2 == 0:` and it will be the right condition for even numbers. Also, you can use here `not` operator: `if not int(enter) % 2:`

Reply:
I just accidentally pushed old file to this branch. I added "==0 condition" as it was before. Code is working now. Check please